sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -259,6 +262,34 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
     }
   }
 
+  /**
+   * Re-register leaks checking hook if configured.
+   */
+  private def ReRegisterCheckLeakHook: Unit = {
```

Comment:
nit : this is a method (should start with a lowercase) and it has side effects, should use empty parameter list in parens. 
```suggestion
  private def reRegisterCheckLeakHook(): Unit = {
```

Reply:
Done.